docker-compose.yml

```diff
@@ -27,17 +34,35 @@ services:
       TMPDIR: /tmp
       # Uncomment the following line for MAX logins
       # MAX_URL: https://login.max.gov/cas/login
+      # The following settings are to maintain environment parity between the
+      # docker-compose configuration for local development and the cloud.gov
+      # staging and production configurations.
       VCAP_APPLICATION: >
         {"uris": ["localhost", "0.0.0.0", "127.0.0.1", "prod-api", "dev-api"]}
       VCAP_SERVICES: >
         {"config": [{"credentials":
           {"DJANGO_SECRET_KEY": "NotASecret", "USING_SSL": "False"}
-        }]}
+        }],
+          "s3": [
+           {
+            "credentials": {
+             "access_key_id": "LOCAL_ID",
+             "bucket": "pdfs",
+             "secret_access_key": "LOCAL_KEY",
+             "region": "irrelevant fake region",
+             "endpoint": "http://minio:9100"
+            },
+            "label": "s3",
+            "name": "storage-s3"
+            }
+           ]
+        }
```

Comment:
Minor, but would you mind cleaning up this JSON so that it and the `config` key above are consistently formatted?